docs/plugin-proposal-private-property-in-object.md

```diff
@@ -0,0 +1,123 @@
+---
+id: babel-plugin-proposal-private-property-in-object
+title: @babel/plugin-proposal-private-property-in-object
+sidebar_label: proposal-private-property-in-object
+---
+
+## Example
+
+**In**
+
+```javascript
+class Foo {
+  #bar = "bar";
+
+  test(obj) {
+    return #bar in obj;
+  }
+}
+```
+
+**Out**
+
+```javascript
+class Foo {
+  constructor() {
+    _bar.set(this, {
+      writable: true,
+      value: "bar"
+    });
+  }
+
+  test() {
+    return _bar.has(this);
+  }
+
+}
+
+var _bar = new WeakMap();
+```
+
+## Installation
+
+```sh
+npm install --save-dev @babel/plugin-property-private-property-in-object
+```
+
+## Usage
+
+### With a configuration file (Recommended)
+
+```json
+{
+  "plugins": ["@babel/plugin-property-private-property-in-object
```

Comment:
```suggestion
  "plugins": ["@babel/plugin-property-private-property-in-object"]
```

Reply:
Lol, I messed up the module name here, too. Will fix in manual commit.